watchdog/src/remote_api/api_bitaps_com.rs

```diff
@@ -0,0 +1,98 @@
+use crate::remote_api::http::{
+    apply_to_body_json, build_transform_context, create_request, fetch_body,
+};
+use crate::remote_api::storage;
+use crate::types::BlockHeight;
+use ic_cdk::api::management_canister::http_request::{
+    CanisterHttpRequestArgument, HttpResponse, TransformArgs,
+};
+use serde_json::json;
+
+/// The transform function for the remote API.
+#[ic_cdk_macros::query]
+fn transform_api_bitaps_com(raw: TransformArgs) -> HttpResponse {
+    apply_to_body_json(raw, ApiBitapsCom::transform)
+}
```

Comment:
nit: (feel to ignore) I personally find it clearer if all of the canister's endpoints are in `main.rs`, so that it's clear to the reader what the canister exposes to the world.